Changes

```diff
@@ -348,6 +348,9 @@ OCaml 5.1.0
 
 ### Tools:
 
+- #11787: Fix GDB scripts to work with OCaml 5's heap layout. (Nick
+  Barnes, review by ???)
```

Comment:
I think you could just remove the "review by ???" field here (no need to list me).